Update running instructions (#3449)
prestosql/presto: README.md

## Requirements

* Mac OS X or Linux

> * Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended and will soon become required. Both Oracle JDK and OpenJDK are supported.

Reviewer: *is the minimum supported ...*

> * Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended although it's not required yet. Both Oracle JDK and OpenJDK are supported.

Reviewer 2: ```suggestion
* Java 8 Update 161 or higher (8u161+), 64-bit is minimal supported version. Java 11 is recommended and will
  soon become required.
```

Presto comes with sample configuration that should work out-of-the-box for development. Use the following options to create a run configuration:

* Main Class: `io.prestosql.server.PrestoServer`

> * VM Options: `-ea -XX:+UseG1GC -XX:G1HeapRegionSize=32M -XX:+UseGCOverheadLimit -XX:+ExplicitGCInvokesConcurrent -Xmx2G -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties -Djdk.attach.allowAttachSelf=true -Dpresto-temporarily-allow-java8=true`

Reviewer 3: Shouldn't we change the paragraph above to state that one should use JDK 11 instead? (language level may still stay at 8.0)

Author: to be honest, idk, let's ask @findepi

Reviewer: Yes, this should talk about running with Java 11. We're going to make that a requirement soon.

Author: @martint @losipiuk take a look now

    -Dhive.metastore.uri=thrift://localhost:9083

> When running on JDK8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.    

Reviewer 2: ```suggestion
When running on Java 8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.    
```

> When running on Java 8 `VM Options` must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order to Presto to start.    

Reviewer 2: ```suggestion
When running on Java 8, the VM Options must be supplemented with `-Dpresto-temporarily-allow-java8=true` in order for Presto to start.
```
